common/src/main/java/fudge/notenoughcrashes/stacktrace/ModIdentifier.java

```diff
@@ -48,9 +58,18 @@ private static Set<CommonModMetadata> identifyFromThrowable(Throwable e) {
         ModsByLocation modMap = NecPlatform.instance().getModsAtLocationsInDisk();
 
         Set<String> involvedClasses = new LinkedHashSet<>();
+        Set<IMixinInfo> involvedMixins = new LinkedHashSet<>();
         while (e != null) {
             for (StackTraceElement element : e.getStackTrace()) {
                 involvedClasses.add(element.getClassName());
+                MixinMerged mixinMerged = findMixinMerged(element);
```

Comment:
Refactor these new lines into a new method `IMixinInfo getMixinInfo(StackTraceElement element)`
and then do `involvedMixins.add(getMixinInfo(element))`